odoo-shippable/Dockerfile

```diff
@@ -106,8 +106,9 @@ RUN pip install pylint_odoo
 #  Setting global env for next shippable build
 ENV WITHOUT_DEPENDENCIES 1
 
-# Install curl standard package to download files
-RUN apt-get install curl
+# Install standard packages
+RUN apt-get install -y curl htop p7zip-full pgbadger \
```

Comment:
@moylop260 Curl is already installed in the base ubuntu image: https://github.com/Vauxoo/docker-ubuntu-base/blob/master/Dockerfile#L23

(sorry for my late reply)